cmd/collector/app/flags/flags.go

```diff
@@ -30,10 +30,11 @@ import (
 )
 
 const (
-	flagDynQueueSizeMemory = "collector.queue-size-memory"
-	flagNumWorkers         = "collector.num-workers"
-	flagQueueSize          = "collector.queue-size"
-	flagCollectorTags      = "collector.tags"
+	flagDynQueueSizeMemory          = "collector.queue-size-memory"
+	flagNumWorkers                  = "collector.num-workers"
+	flagQueueSize                   = "collector.queue-size"
+	flagCollectorTags               = "collector.tags"
+	flagProcessedSpanMetricsEnabled = "collector.enable-span-size-metrics"
```

Comment:
```suggestion
	flag SpanSizeMetricsEnabled = "collector.enable-span-size-metrics"
```
let's keep consistent naming

Reply:
oh ok, I will check it all again